Admin/MenuNodeAdmin.php

```diff
@@ -160,10 +160,10 @@ public function getFormBuilder()
      */
     public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
     {
-        $menuNodeNode = parent::buildBreadcrumbs($action, $menu);
+        $menuNodeItem = parent::buildBreadcrumbs($action, $menu);
```

Comment:
to be honest, I don't understand what this method is doing at all. I assume it changes the breadcrumb display for menu nodes, so it would make sense to call it menu node item.